downloads/index.md

```diff
@@ -267,23 +267,20 @@ Most Linux users should use the glibc binaries unless you know that your system
 
 Older releases of Julia for all platforms are available on the [Older releases page](/downloads/oldreleases/). Only the LTS and Stable releases are maintained.
 
-## "Nightly" builds
+## Nightly Builds
 
-Builds of the current unstable development version of Julia are available on the [nightlies page](/downloads/nightlies/). Built after each update to master, these are intended as developer previews into the latest work and are not intended for normal use. Most users are advised to use the current release version of Julia, above.
+Builds of the current unstable development version of Julia are available on the [nightlies page](/downloads/nightlies/). While the name suggests that they are built every night, they are not strictly nightly. At times, due to broken builds or CI infrastructure issues, nightlies may not be produced. These are intended as developer previews into the latest work and are not intended for normal use. Most users are advised to use the current stable release version of Julia.
 
 ## Download verification
 
 All Julia binary releases are cryptographically secured using the traditional methods on each platform. macOS and Windows releases are codesigned with certificates that are verified by the operating system during installation. Linux and source tarballs are signed with GPG using [this key](/assets/juliareleases.asc).
 
-## Platform specific instructions
+---
 
-Please see [platform specific instructions](/downloads/platform/) for further installation instructions and if you have trouble installing Julia.
```

Comment:
Why is this removed? I don't see any other links to these pages

Reply:
I thought this is linked in the help parts above.